src/main/java/org/kuali/kfs/coa/document/validation/impl/AccountGlobalRule.java

```diff
@@ -1607,49 +1584,21 @@ protected boolean checkCloseAccount(AccountGlobalDetail detail) {
 	  * @return fails if the expiration date is null or after today's date
 	  */
 	protected boolean checkAccountExpirationDateValidTodayOrEarlier(CuAccountGlobal newAccount) {
-		// get the expiration date, if any
-		Date expirationDate = newAccount.getAccountExpirationDate();
-		if (ObjectUtils.isNotNull(expirationDate)) {
-
-			Date todaysDate = new Date(getDateTimeService().getCurrentDate().getTime());
-			todaysDate.setTime(DateUtils.truncate(todaysDate,Calendar.DAY_OF_MONTH).getTime());
-
-			// when closing an account, the account expiration date must be the
-			// current date or earlier
-			expirationDate.setTime(DateUtils.truncate(expirationDate, Calendar.DAY_OF_MONTH).getTime());
-			if (expirationDate.after(todaysDate)) {
-				putFieldError(KFSPropertyConstants.ACCOUNT_EXPIRATION_DATE, KFSKeyConstants.ERROR_DOCUMENT_ACCMAINT_ACCT_CANNOT_BE_CLOSED_EXP_DATE_INVALID);
-				return false;
-			}
-		}
-
-		return true;
-	}
-	
-    protected boolean checkAccountExpirationDateValidTodayOrEarlier(AccountGlobalDetail accountGlobalDetail) {
-
         // get today's date, with no time component
         Date todaysDate = new Date(getDateTimeService().getCurrentDate().getTime());
         todaysDate.setTime(DateUtils.truncate(todaysDate, Calendar.DAY_OF_MONTH).getTime());
-        // TODO: convert this to using Wes' Kuali KfsDateUtils once we're using Date's instead of Timestamp
-        
-        accountGlobalDetail.refreshReferenceObject(KFSPropertyConstants.ACCOUNT);
-        Account existingAccount = accountGlobalDetail.getAccount();
 
         // get the expiration date, if any
-        Date expirationDate = existingAccount.getAccountExpirationDate();
+        Date expirationDate = newAccount.getAccountExpirationDate();
```

Comment:
If the AccountGlobal expiration date field is empty and the close account field is yes then the user should be able to still close the account if the account being edited already has an expiration date on it. I am afraid after this change the user is always required to enter an expiration date on account global doc when closing an account.